rotkehlchen/tasks/manager.py

```diff
@@ -48,7 +48,7 @@
 TX_DECODING_LIMIT = 500
 
 
-def noop_exchange_succes_cb(trades, margin, asset_movements, ledger_actions, exchange_specific_data) -> None:  # type: ignore # noqa: E501
+def noop_exchange_succes_cb(trades, margin, asset_movements, exchange_specific_data) -> None:  # type: ignore # noqa: E501
```

Comment:
Perhaps you can also fix my spelling :)


```suggestion
def noop_exchange_success_cb(trades, margin, asset_movements, exchange_specific_data) -> None:  # type: ignore # noqa: E501
```